feg/gateway/docker/go/Dockerfile

```diff
@@ -61,7 +61,7 @@ ENV GOBIN /var/opt/magma/bin
 ENV MAGMA_ROOT /magma
 ENV PIP_CACHE_HOME ~/.pipcache
 ENV PYTHON_BUILD /build/python
-ENV PATH ${PYTHON_BUILD}/bin:${PATH}:${GOBIN}
+ENV PATH ${PYTHON_BUILD}/bin:${PATH}:${GOBIN}:/usr/local/go/bin
```

Comment:
Why did this path need to be updated?

Reply:
In Go 1.19, the command `go fmt` utilises now `gofmt` which is an executable binary installed in `/usr/local/go/bin`.

Reply:
Wasn't this the case for 1.18 too? When using go 1.18, if I do `go help fmt`, it tells me that it uses `gofmt`:

`Fmt runs the command 'gofmt -l -w' on the packages named`

and I find `gofmt` in the same place: `/usr/local/go/bin/gofmt`

Reply:
But without the go path in `PATH`, the precommit check fails with:
```
gofmt -l -s -w .
make: gofmt: Command not found
make: *** [Makefile:45: fmt] Error 127
```

Reply:
For consistency, I would vote for symlinking it like is done for go, but I don't have a strong preference